README.md

```diff
@@ -48,19 +49,31 @@ $ lb app loopback-example-access-control
 ... # follow the prompts
 $ cd loopback-example-access-control
 ```
+### Add the datasource
+- Name: `mycloudant`
+  - datasource: `cloudant`
+  - url: `your_cloudant_url`
+  - database: `your_cloudant_database`
+  - username: `your_cloudant_username`
```

Comment:
do we need username and password?  i thought it's already sufficient if we have url.